src/main/java/com/MyApp/budgetControl/api/GlobalExceptionHandler.java

```diff
@@ -39,7 +40,14 @@ class GlobalExceptionHandler {
   @ResponseStatus(HttpStatus.NOT_FOUND)
   @ExceptionHandler(value = NoSuchElementException.class)
   public @ResponseBody ErrorResponse handleNoSuchElementExceptions(NoSuchElementException ex) {
-    List<String> errorMessage = List.of("Expense with given Id does not exist");
+    List<String> errorMessage = List.of("Element with given Id does not exist");
```

Comment:
For such hardcoded strings its a good practise to declare them as static fields on class level